Libraries/src/Amazon.Lambda.AppSyncEvents/README.md

```diff
@@ -0,0 +1,32 @@
+# Amazon.Lambda.AppSyncEvents
+
+This package contains classes that can be used as input types for Lambda functions that process AppSync events.
+
+# Sample Function
+
+The following is a sample class and Lambda function that receives AppSync event record data as an `appSyncResolverEvent` and logs some of the incoming event data. (Note that by default anything written to Console will be logged as CloudWatch Logs events.)
```

Comment:
Please change `AppSync` to `AppSync resolver`

Reply:
Updated